cloud/amazon/ec2_vol.py

```diff
@@ -100,6 +100,18 @@
     default: present
     choices: ['absent', 'present', 'list']
     version_added: "1.6"
+  kms_key_id:
+    description:
+      - full arn of the kms key id is required.
+        e.g.: arn:aws:kms:us-east-1:012345678910:key/abcd1234-a123-456a-a12b-a123b4cd56ef
+    required: false
+    default: None
+    version_added: 2.39
```

Comment:
This should be the ansible version it got merged in under, not the boto version. Change this to 2.2 for example as thats likely the version this would get merged into.